node/src/GlideClusterClient.ts

```diff
@@ -970,16 +1029,32 @@ export class GlideClusterClient extends BaseClient {
     public async functionList(
         options?: FunctionListOptions & DecoderOption & RouteOption,
     ): Promise<ClusterResponse<FunctionListResponse>> {
-        return this.createWritePromise(createFunctionList(options), {
+        return this.createWritePromise<
+            GlideRecord<unknown> | GlideRecord<unknown>[]
```

Comment:
It is intermediate type, I put `unknown`, because I'm too lazy to put the correct type there. The output type is well-defined, see line 1031 - `ClusterResponse<FunctionListResponse>`.
Also linter blames `any`, but accepts `unknown`.